src/Modules/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/profileSettings/profileProperties/profilePropertyEditor/index.jsx

```diff
@@ -117,16 +118,16 @@ class ProfilePropertyEditor extends Component {
     }
 
     isValidName(name) {
-        const validatePropertyName = /^[a-zA-Z0-9]+$/g;
-        const isValid = validatePropertyName.test(name);
+        const validatePropertyName = /^\w*[a-zA-Z0-9]+$/g;
+        const isValid = (name) ? validatePropertyName.test(name) : false;
```

Comment:
Good catch with the ternary. I wouldn't have expected that `null`/`undefined` actually return `true` 😮